sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -338,7 +338,7 @@ def start_bundle(self):
 
     self._counter_factory = counters.CounterFactory()
 
-    dofn = copy.deepcopy(transform.dofn)
+    dofn = pickler.loads(pickler.dumps(transform.dofn))
```

Comment:
Can you add a todo here to perhaps store this in serialized form?  This seems suboptimal but all right for now and not yet worth changing.